.github/workflows/push-to-aws-s3.yaml

```diff
@@ -0,0 +1,66 @@
+name: Push to S3 bucket
+
+on:
+  push:
+    tags:
+      - "v[0-9]+"
+      - "v[0-9]+-*"
+      - "v[0-9]+.[0-9]+"
+      - "v[0-9]+.[0-9]+-*"
+      - "v[0-9]+.[0-9]+.[0-9]+"
+      - "v[0-9]+.[0-9]+.[0-9]+-*"
+    branches:
+      - "**"
+
+jobs:
+  s3:
+    runs-on: ubuntu-latest
+
+    permissions:
+      id-token: write
+      contents: read
+
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Configure AWS Credentials
+        uses: aws-actions/configure-aws-credentials@v4
+        with:
+          role-to-assume: arn:aws:iam::${{ secrets.AWS_ACCOUNT_ID }}:role/github-actions-container-registry
+          aws-region: ${{ secrets.AWS_REGION }}
+
+      - name: Get image tag
+        run: echo IMAGE_TAG=$(if [[ "${GITHUB_REF##refs/tags/}" =~ ^v ]]; then echo ${GITHUB_REF##refs/tags/}; else echo "${GITHUB_REF##refs/heads/}"; fi) >> "$GITHUB_ENV"
+
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v4
+        with:
+          node-version: ${{ matrix.node-version }}
+
+      - name: Build Node files
+        run: |
+          npm install mini-css-extract-plugin 
+          npm install css-loader 
+          npm install crypto-browserify
+          npm install stream-browserify
+          npm install browserify-zlib
+          npm run build
+          cd dist/assets
+          for ext in js css; do
+              for file in index-*.$ext; do
+                  mv "$file" "index.$ext"
+                  echo "Renamed $file to index.$ext"
+              done
+          done
```

Comment:
I believe it's got something to do with the fact that we are importing the `index.js` file from S3 using a `<script>` tag, so otherwise it wouldn't find it.